tests/python/unittest/test_arith_simplify.py

```diff
@@ -29,6 +29,9 @@ def test_simplify():
     #  assert tvm.ir_pass.Equal(tvm.ir_pass.CanonicalSimplify(n / (-1)),
     #                           tvm.ir_pass.CanonicalSimplify(-n))
 
+def test_simplify_div():
+    x = tvm.var('x')
+    assert tvm.ir_pass.CanonicalSimplify((241+48*x)/16 - (15 + (x*3))).value == 0
```

Comment:
could you also add a non-divisible test case, e.g., ((241 + (x*15))/4)

Reply:
done.